docs/conf_sso_test_msscolab.rst

```diff
@@ -1,6 +1,6 @@
 Configuration MSS Colab Server with Testing IdP for SSO
 =======================================================
-Testing IDP (`mslib/msidp`) is specifically designed for testing the Single Sign-On (SSO) process with the mscolab server using PySAML2.
+Testing IDP (`mslib/msidp`) is specifically designed for testing the Single Sign-On (SSO) process with the mscolab server using PySAML2. This is solely for development and testing purposes, do not use in production environments.
 
```

Comment:
use the caution or warning attribute, https://sphinxawesome.xyz/demo/notes/

That hightlights it better.

e.g. 
```
.. warning::
   When running publicly rather than in development, you should not use the built-in development server ( msidp / idp.py ).

   The development server is provided by MSS for convenience, but is not designed to be particularly efficient, stable, or secure.
```